translations/de/taboo.json

```diff
@@ -69,8 +67,7 @@
     "octgn_id": "3d114b67-5461-4acf-9cb3-3dbd054d5975",
     "url": "https://de.arkhamdb.com/card/02108",
     "imagesrc": "/bundles/cards/02108.png",
-    "taboo_xp": 2,
-    "WARNING": "NOT UPDATED"
+    "taboo_xp": 2
```

Comment:
I think these "taboo_xp" text can be deleted as well.

Reply:
Alright, removed it